src/main/scala/sigmastate/types.scala

```diff
@@ -110,7 +111,7 @@ object SType {
     * NOTE: in the current implementation only monomorphic methods are supported (without type parameters)*/
   val types: Map[Byte, STypeCompanion] = Seq(
     SNumericType, SString, STuple, SGroupElement, SSigmaProp, SContext,
-    SAvlTree, SBox, SOption, SCollection
+    SAvlTree, SBox, SOption, SCollection, SBigInt, SOption
```

Comment:
Why SOption is two times in this list?

Reply:
Merging artifacts I suppose. Thanks for spotting this one! Fixed.